google-cloud-core/pom.xml

```diff
@@ -101,22 +101,22 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
-      <version>3.0.0-beta-3</version>
+      <version>3.0.0</version>
```

Comment:
I see that 2 more recent versions already exist for `protobuf-java`.
- [3.0.2](https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-java/3.0.2/)
- [3.1.0](https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-java/3.1.0/)

Might be worth jumping straight to `3.1.0`. I run integration tests with no ugly surprises.